.travis.yml

```diff
@@ -15,6 +15,11 @@ matrix:
       osx_image: beta-xcode6.2
       rvm: system
 
+# Remove Xcode and CommandLineTools included in default OS X image.
+# This ensures the Command Line Tool install functionality is tested.
+before_script:
+  - sudo rm -rf /Applications/Xcode.app /Library/Developer/CommandLineTools
```

Comment:
To be sure, this one? 

https://github.com/mikemcquaid/strap/blob/78e162d5d239d1bfb5788d4dd9fb8bb7222fac13/.travis.yml#L20